src/Umbraco.Web/PublishedCache/PublishedMember.cs

```diff
@@ -140,9 +140,11 @@ private void EnsureMemberProperty(List<IPublishedProperty> properties, List<stri
         public override string UrlSegment => throw new NotSupportedException();
 
         // TODO: ARGH! need to fix this - this is not good because it uses ApplicationContext.Current
+        [Obsolete("Use WriterName(IUserService) extension instead")]
         public override string WriterName => _member.GetCreatorProfile().Name;
 
         // TODO: ARGH! need to fix this - this is not good because it uses ApplicationContext.Current
```

Comment:
Stuff like this highlights to me that these need refactoring because they've proven tough to implement nicely in the past